projects/angular/src/layout/main-container/API.md

```diff
@@ -0,0 +1,34 @@
+# Header-clarity - Style API
+
+| CSS Variable Name          | Description                                                        |
+| -------------------------- | ------------------------------------------------------------------ |
+| --clr-header-bg-color | Changes the background color of the first header in the header's colors.
+| --clr-header-divider-opacity | Changes the transparency of the header element
+| --clr-header-nav-opacity | Changes the header's transparency and applies the change to the top header of the page.
+| --clr-header-nav-hover-opacity | changes the header's transparency when the mouse is over it.
+| --clr-header-2-bg-color | Changes the background color of the second header in the header's colors.
+| --clr-header-3-bg-color | Changes the background color of the third header in the header's colors.
```

Comment:
we don't need "in the header's colors"
it should be applies for all similar lines below...

Reply:
Removed, As suggested